components/backstage/components/backstage_sidebar.jsx

```diff
@@ -150,28 +157,23 @@ export default class BackstageSidebar extends React.Component {
         );
 
         return (
-            <TeamPermissionGate
-                permissions={[Permissions.MANAGE_INCOMING_WEBHOOKS, Permissions.MANAGE_OUTGOING_WEBHOOKS, Permissions.MANAGE_SLASH_COMMANDS, Permissions.MANAGE_OAUTH]}
-                teamId={this.props.team.id}
+            <BackstageCategory
```

Comment:
Can't use any of the `*Gate` tools here, since we need to test for both a team permission and a system permission. This is now handled by `canManageIntegrations`.